src/common/lib/types/basemessage.ts

```
@@ -0,0 +1,270 @@
+import * as Utils from '../util/utils';
+import { Bufferlike as BrowserBufferlike } from '../../../platform/web/lib/util/bufferutils';
+import * as API from '../../../../ably';
+import { actions } from './protocolmessagecommon';
+
+import type { IUntypedCryptoStatic } from 'common/types/ICryptoStatic';
+import type { ChannelOptions } from '../../types/channel';
+import type ProtocolMessage from './protocolmessage';
+
+export type CipherOptions = {
+  channelCipher: {
+    encrypt: Function;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)